README.md

```diff
@@ -17,7 +17,9 @@ This is a react web app where users can look up and see details about their coun
 
 ## Demo Video
 
-[Video-Demo] (https://www.loom.com/share/77c88a658fe444fe9854242fbefd0d1a/)
+
+[Demo Video](https://www.loom.com/share/77c88a658fe444fe9854242fbefd0d1a/)
+
 
 
 
```

Comment:
- [ ] Please add some steps to the Getting Started section in your readme on how the users can run your project locally (ex: clone project, cd project, npm install, npm start ...)